src/components/Search/SearchResultSeries.js

```diff
@@ -49,14 +49,14 @@ class SearchResultSeries extends SearchResultBase {
     };
   };
 
+  // Reduce series from all leaf tags/sources.
   getLowestLevelSeries = (series, rootId) => {
-    if (series.length <= 1)
-      return series[0];
+    if (!series || !series.length) return null;
     const root = series.find(s => s.parent_id === rootId || (!s.parent_id && !rootId));
-    if (!(root?.children?.length !== 0)) {
```

Comment:
this is not same?

Reply:
Not sure what is root is undefined, then !undefined is true...
But in my case it is false.

Reply:
I hate those ?. operators... they are not readable....
Please use positive logic and not negative.